pkg/filetree/loader.go

```diff
@@ -85,18 +86,29 @@ func (a *aferoLoader) LoadTree(root string) (*Node, error) {
 		Name:     "/",
 		Children: []Node{},
 	}
-	overlayRootNode := Node{
+	patchesRootNode := Node{
 		Path:     "/",
-		Name:     OverlaysFolder,
+		Name:     PatchesFolder,
 		Children: []Node{},
 	}
+	resourceRootNode := Node{
+		Path:     "/",
+		Name:     ResourcesFolder,
+		Children: []Node{},
+	}
+
+	populatedBase, err := a.loadTree(fs, rootNode, files)
```

Comment:
what happens to this error 😢

Reply:
oh good call

Reply:
looks like it was never being handled?

Reply:
wait - no -it happens up here https://github.com/replicatedhq/ship/pull/639/files#diff-054ba8f349d9b1eadd46df29c7372a83L106

Reply:
but that gap is getting pretty wide and we probably ought to just `!= nil` it instead